graylog2-server/src/main/java/org/graylog2/security/ldap/LdapSettingsImpl.java

```diff
@@ -218,13 +238,34 @@ public boolean isActiveDirectory() {
 
     @Override
     public String getDefaultGroup() {
+        final String defaultGroupId = getDefaultGroupId();
+        if (defaultGroupId.equals(roleService.getReaderRoleObjectId())) {
+            return "Reader";
```

Comment:
Maybe put this into a constant and use it in all relevant places instead of the plain string, which is prone to typos.